README.rst

```diff
@@ -9,6 +9,10 @@ Weaviate python client
     :target: https://travis-ci.com/semi-technologies/weaviate-python-client
```

Comment:
We do not need the Travis Badge anymore.

Reply:
gone